integration-tests/steps/helpers.py

```diff
@@ -47,6 +48,8 @@ def step_impl(context, exception_type):
         assert isinstance(context.error, AttributeError)
     elif exception_type == "UndefinedError":
         assert isinstance(context.error, jinja2.exceptions.UndefinedError)
+    elif exception_type == "SceptreException":
+        assert isinstance(context.error, SceptreException)
```

Comment:
This whole test is nuts and I'd be more in favour of just removing the test so no one else has to waste time on this.

Reply:
we are open to the idea of removing the test but only if you can provide a good reason.

Reply:
It's just a badly designed test that isn't testing anything useful as far as I can tell. However, not worth discussing too much!